supervisor/api/hardware.py

```diff
@@ -58,8 +58,3 @@ async def audio(self, request: web.Request) -> Dict[str, Any]:
                 },
             }
         }
-
-    @api_process
-    async def trigger(self, request: web.Request) -> Awaitable[None]:
-        """Trigger a udev device reload."""
-        _LOGGER.debug("Ignoring DEPRECATED hardware trigger function call.")
```

Comment:
Instead of removing this.
Should we change back the level to the warning, and include the "request_from" ?

Reply:
This feature was never real in the wild, It was just used for deconz and worked too bad for going forward that, we replaced it with a new hardware layer which also forced the user to update deconz (because of a mistake) and on that update, we removed that call.

Reply:
Alright 👍